### PR TITLE
improved documentation of siminfomanager

### LINKED ISSUE #956: add more climate data
http://climate.onebuilding.org
contains tons of TMY files. We could test compatibility and at least integrate a link to this page in the documentation.

### PR DESCRIPTION
**Pull request by Mathadon**
this closes #956

### CODE REVIEW

**Comment by damienpicard** [comment]
@Mathadon have you tested the compatibility?

**Comment by Mathadon** [comment]
@damienpicard I didn't

**Comment by Mathadon** [comment]
@damienpicard ok to merge this regardless?

**Comment by damienpicard** [comment]
How about removing the reference to the website containing TMY if we do not find the time to test it?

File: IDEAS/BoundaryConditions/SimInfoManager.mo [hunk 1]
```diff
@@ -98,9 +98,20 @@ This should only be done if a custom weather data reader instead
 of the TMY3 weather data reader is used.
 </li>
 </ul>
+<h4>TMY3 weather data files</h4>
+IDEAS uses TMY3 input files. For detailed documentation see 
+<a href=\"modelica://IDEAS.BoundaryConditions.WeatherData.ReaderTMY3\">IDEAS.BoundaryConditions.WeatherData.ReaderTMY3</a>.
+See <a href=\"http://climate.onebuilding.org\">http://climate.onebuilding.org</a> for more climate file examples.
```
Comment (Mathadon) [inline IDEAS/BoundaryConditions/SimInfoManager.mo]:
```suggestion
```

**Comment by Mathadon** [comment]
ok fixed in a55d07c

**Pull request merged**